ukvm/ukvm_hv_freebsd_x86_64.c

```diff
@@ -187,9 +187,8 @@ void ukvm_hv_vcpu_loop(struct ukvm_hv *hv)
 
         switch (vme->exitcode) {
         case VM_EXITCODE_SUSPENDED:
```

Comment:
Please remove this entire `case` block, so that execution falls through to `default:` below.

Reply:
removed